qshift/templates/quarkus-application/manifests/argocd/${{values.component_id}}-db.yaml

```diff
@@ -0,0 +1,42 @@
+{% if values.database == 'quarkus-jdbc-postgresql' %}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: ${{ values.component_id }}-db
+  namespace: ${{ values.gitopsNamespace }}
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: ${{ values.appNamespace }}
+  source:
+    chart: postgresql
+    repoURL: https://charts.bitnami.com/bitnami
+    targetRevision: 11.9.13
+    helm:
+      values: |
+        auth:
+          username: ${{ values.database_username }}
+          password: ${{ values.database_password }}
```

Comment:
we should probably not pass the password here and let helm generate one.

Reply:
This is correct. I missed to remove it here ;-). Will fix it